docs/endpoints.rst

```diff
@@ -1030,7 +1030,9 @@ Extraction Endpoints
 
    :query time: required; format same as described in time_
    :query properties: specifies what properties should be included for each feature representing an OSM element: ‘tags’ and/or 'metadata’; multiple values can be delimited by commas; default: empty
+   :query unclipped: boolean operator to apply a clipping to the geometries of the returned featues (‘true’), or not (‘false’); default: ‘false’
```

Comment:
I'm wondering if this is clear enough… maybe we can tell what the geometries will be clipped against?

Reply:
You mean something like ".. apply a clipping on the returned features using the defined spatial boundary..."?

Reply:
i would have maybe written something like: _“boolean operator to specify whether the returned features' geometries should be clipped to the query's spatial boundary (‘false’), or not (‘true’); default: ‘false’”_

Reply:
PS: the values `true`/`false` are reversed in the description above: `true` would be for unclipped features, while `false` returns the clipped ones.